content/blogs/release-0-22.md

```diff
@@ -0,0 +1,381 @@
+
+For detailed information on setting up and configuring LDAP in Kestra, check our [LDAP documentation](/docs/enterprise/auth/sso/ldap).
+
+![ldap](/blogs/release-0-22/ldap.png)
+
+### New Log Shipper plugins
+
+This release adds a new `AuditLogShipper` and new log exporters plugins to the Enterprise Edition, including:
+- Splunk
+- OpenSearch
+- AWS S3
+- Google Cloud Storage
+- Azure Blob Storage.
+
+The cloud storage log exporters provide a cost-effective long-term storage solution for your logs.
+
+Additionally, the new `AuditLogShipper` plugin allows you to export audit trails to multiple destinations, providing a convenient way to analyze comprehensive record of all user and service account actions within your Kestra instance.
+
+For detailed information on setting up and configuring log shippers, check the [Log Shipper documentation](https://kestra.io/docs/enterprise/governance/logshipper).
+
+### Unified Secrets and KV Store UI
+
+This release introduces new global views for managing secrets and key-value pairs across your entire Kestra instance:
+
+- **KV Store UI:** We've added a dedicated page listing all key-value pairs across all namespaces.
+- **Secrets UI:** Enterprise Edition users gain a unified view of all Secrets existing in your instance across all namespaces, simplifying governance and bringing visibility about which secrets exist and within which namespaces they are managed.
+
+
+## Other Features and Improvements
+
+- We've revamped our **Queues** for performance and reliability. You can expect the `queues` database table to take up less space and perform better
+- [DevContainer support](docs/01.getting-started/03.contributing.md) simplifies development setup for contributors with ready-to-use environments
+- [New Python package](https://github.com/kestra-io/libs/pull/16) allow you to read Kestra's native ION files into Pandas or Polars dataframes. Read more in our [Python How-to guide](/docs/how-to-guides/python)
+- Improved Ansible integration with the ability to [capture outputs from individual steps](https://github.com/kestra-io/plugin-ansible/pull/35),of your Ansible playbooks
+- Multiple bug fixes for dynamic properties ensure more reliable and predictable behavior across workflows
+- Website performance dramatically improved following Nuxt 2 to 3 migration, including a redesigned plugin page for easier navigation of plugin properties and outputs
+- Expanded context variables now include [taskrun and execution states accessible via Pebble](https://github.com/kestra-io/kestra/issues/7155). The `{{tasks.your_task_id.state }}` context returns a task run's state while the `{{execution.state}}` allows to retrieve the flow execution state.
```

Comment:
```suggestion
- Expanded context variables now include [taskrun and execution states accessible via Pebble](https://github.com/kestra-io/kestra/issues/7155). The `{{ tasks.your_task_id.state }}` context returns a task run's state while the `{{ execution.state }}` allows to retrieve the flow execution state.
```

Reply:
looks strange to add spaces in simple inline text, even in flows we now avoid extra spaces + no need to be that rigid about this, in the end both work :)